### DISCUSSION TITLE
Created at & Updated at from client

### DISCUSSION THREAD

**Discussion by fbede** [pocketbase/pocketbase#3284] [Q&A] [answered]
Is it possible to update the default "created at" and "updated at" timestamp fields from the client?

My use case is an offline first app with an embedded database and I would like the "created at" & "updated at" fields in some tables to reflect when the row was created or updated on the client rather than on the server.

**Comment by ganigeorgiev** [comment, marked as answer]
At the moment it is not possible, at least not out of the box.
You can create your own `date` fields or override the default `created`/`updated` fields value in a server hook as shown in https://github.com/pocketbase/pocketbase/discussions/3248.
